source/rss-qt-node/rss_feed.js

```diff
@@ -0,0 +1,50 @@
+var cppDemoModule = process.binding('CppDemoModule');
+
+var FeedParser = require('feedparser');
+var request = require('request'); // for fetching the feed
+
+var emitRequest = function () {
+  console.log("Refreshing feeds...")
+  var feedparser = new FeedParser([]);  
+  var req = request('http://feeds.bbci.co.uk/news/world/rss.xml')
+
+  req.on('error', function (error) {
+    // handle any request errors
+  });
+
+  req.on('response', function (res) {
+    var stream = this; // `this` is `req`, which is a stream
+  
+    if (res.statusCode !== 200) {
+      this.emit('error', new Error('Bad status code'));
+    }
+    else {
+      cppDemoModule.clearFeed();
+      stream.pipe(feedparser);
+    }
+  });
+ 
+  feedparser.on('error', function (error) {
+    // always handle errors
```

Comment:
The comment is a bit misleading, isn't it?

Reply:
yeah. should probably be something like "catch all parser errors but don't handle them"

Reply:
fixed in 77d6b2e.